src/components/download-track-progress-drawer/DownloadTrackProgressDrawer.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useCallback } from 'react'
+
+import LottieView from 'lottie-react-native'
+import { Platform, StyleSheet, View } from 'react-native'
+import { useSelector } from 'react-redux'
+
+import Drawer from 'app/components/drawer'
+import Text from 'app/components/text'
+import { useDrawer } from 'app/hooks/useDrawer'
+import {
+  getDownloadedPercentage,
+  getFileName,
+  getFetchTask
+} from 'app/store/download/selectors'
+
+const styles = StyleSheet.create({
+  view: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    paddingTop: 16,
+    paddingHorizontal: 24
+  },
+
+  title: {
+    fontFamily: 'AvenirNextLTPro-Bold',
+    fontSize: 21,
+    color: '#7E1BCC',
+    marginBottom: 28
+  },
+
+  fileName: {
+    fontFamily: 'AvenirNextLTPro-Bold',
+    fontSize: 18,
+    lineHeight: 24,
+    color: '#C2C0CC',
+    marginBottom: 12,
+    textAlign: 'center'
+  },
+
+  downloadContainer: {
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+    marginBottom: 32
+  },
+
+  loadingIcon: {
+    width: 40,
+    height: 40,
+    alignSelf: 'flex-start',
+    marginRight: 10
+  },
+
+  downloadPercent: {
+    fontFamily: 'AvenirNextLTPro-Heavy',
```

Comment:
More of a preference but if you use our `Text` component you can specify the weight prop and don't have to type out the font family everywhere